_DeploymentAndDistroPackaging/ansible/cleanup/ciao.yml

```diff
@@ -39,12 +39,13 @@
         - ciao-keystone
 
     - name: Stop and disable services
-      service: name={{ item }} state=stopped enabled=no
+      systemd: name={{ item }} state=stopped enabled=no
       with_items:
         - docker-keystone.service
         - docker-ciao-webui.service
         - ciao-controller.service
         - ciao-scheduler.service
+      ignore_errors: yes
```

Comment:
if `ciao-scheduler.service` exists, then ansible makes sure the service is disabled and stopped.

If the file does not exists. Ansible fails to make sure the service is disabled and stopped.

`ignore_errors: yes` let ansible continue with the other tasks as the service is in the desired state (disabled and stopped) when the service does not exist.